src/go/rpk/pkg/cli/topic/consume.go

```diff
@@ -152,22 +168,25 @@ func newConsumeCommand(fs afero.Fs, p *config.Params) *cobra.Command {
 
 	cmd.Flags().StringVar(&c.rack, "rack", "", "Rack to use for consuming, which opts into follower fetching")
 
+	cmd.Flags().StringSliceVar(&c.useSchemaRegistry, "use-schema-registry", []string{}, "If present, rpk will decode the key and the value with the schema registry. Also accepts use-schema-registry=key or use-schema-registry=value")
```

Comment:
I think this should be StringArrayVar, since handling a slice here is a bit odd? This will change the decoding below, though, to require splitting by comma

Reply:
Eh not important and I was confusing ArrayVar and SliceVar -- I meant StringVar directly to avoid any confusion about repeating the flag, but it's not important.